Makefile

```diff
@@ -1,31 +1,31 @@
 run-hello-world-attest-fn-call:
 	@echo "Running run-hello-world-attest-fn-call..."
-	@make -C hello_world_attest_fn_call clean run > /dev/null 2>&1
```

Comment:
Cleaning before every test made the overall testing very time consuming. I removed it not that `run` has udpated dependencies that should trigger rebuild if necessary